mappings/net/minecraft/world/GameRules.mapping

```diff
@@ -1,5 +1,45 @@
 CLASS net/minecraft/class_1928 net/minecraft/world/GameRules
+	FIELD field_19387 Lnet/minecraft/class_1928$class_4313;
+	FIELD field_19388 Lnet/minecraft/class_1928$class_4313;
+		COMMENT A {@linkplain GameRule} which regulates whether a Mob's action can modify the world.
+		COMMENT
+		COMMENT <p>This includes:
+		COMMENT <ul>
+		COMMENT <li>Whether explosions from a Creeper destory blocks
+		COMMENT <li>Whether a zombie can break down a door
+		COMMENT <li>Whether a wither killing an entity will place or drop a wither rose.
+		COMMENT </ul>
+	FIELD field_19389 Lnet/minecraft/class_1928$class_4313;
+		COMMENT A {@linkplain GameRule} which regulates whether a player should keep their inventory's contents upon death.
+	FIELD field_19390 Lnet/minecraft/class_1928$class_4313;
+		COMMENT A {@linkplain GameRule} which regulates whether mobs should spawn naturally.
+	FIELD field_19391 Lnet/minecraft/class_1928$class_4313;
+		COMMENT A {@linkplain GameRule} which regulates whether a mob should drop items.
+	FIELD field_19392 Lnet/minecraft/class_1928$class_4313;
+		COMMENT A {@linkplain GameRule} which regulates whether items should be dropped when a block is broken.
+	FIELD field_19398 Lnet/minecraft/class_1928$class_4313;
+	FIELD field_19401 Lnet/minecraft/class_1928$class_4313;
+		COMMENT A {@linkplain GameRule} which regulates whether clients should use reduced debug info.
+		COMMENT
+		COMMENT <p>When the value of this rule is changed, all connected clients will be notified how much debug info to display.
+		COMMENT This includes the visiblity of coordinates on the client in the DebugHud.
+	FIELD field_19402 Lnet/minecraft/class_1928$class_4313;
+	FIELD field_19405 Lnet/minecraft/class_1928$class_4313;
+		COMMENT A {@linkplain GameRule} which regulates how many entities must be crammed into a block before cramming damage occurs.
+	FIELD field_19409 Lnet/minecraft/class_1928$class_4313;
+		COMMENT A {@linkplain GameRule} which regulates whether a player recieving an advancement should be announced in chat.
 	FIELD field_19410 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_19422 Lnet/minecraft/class_1928$class_4313;
+		COMMENT A {@linkplain GameRule} which regulates whether raids should occur.
```

Comment:
You should use `{@link}` here instead of `{@linkplain}` as they are code names, not plain text. You don't call them "GameRules" but "game rules" when you're not talking about the class.

Reply:
Or `{@linkplain GameRule game rule}`